content/docs/concepts/csidriver/features/powermax.md

```diff
@@ -680,3 +680,90 @@ These are the Container Storage Modules not supported with NVMeTCP protocol:
 - Container Storage Modules Observability
 - Container Storage Modules Application Mobility
 - Metro Replication
+
+## Multiple Avalabilty Zones
+
+Starting with CSM 2.14.0, the PowerMax CSI driver supports multiple availability zones (AZ) for block protocols. NFS is not supported at this time.
+
+This feature supports the use of a StorageClass that is not associated with any specific PowerMax array or storage resource pool (SRP). Each cluster node must be labelled with topology labels which match the labels in the secret.
+
+Requirements:
+- Only one PowerMax array per availability zone is supported.
+- Every PowerMax array must be labelled with one or more zone labels.
+- Every cluster worker node must be assigned to a zone.
+- The StorageClass does not contain any references to the array ID.
+- The volumeBindingMode must be set to WaitForFirstConsumer.
```

Comment:
Is this a hard requirement?

Reply:
Not sure I understand this, are we yet to validate all the requirements listed? In that case please leave them out until they are confirmed. We can follow up with further updates.